java/core/src/test/java/sleeper/core/statestore/inmemory/InMemoryFileInfoStore.java

```diff
@@ -155,16 +142,7 @@ private List<String> findFilenamesWithJobIdSet(List<FileInfo> fileInfos) {
 
     @Override
     public void deleteReadyForGCFile(FileInfo fileInfo) throws StateStoreException {
```

Comment:
We should be able to delete this method signature, since deleteReadyForGCFile(String) should be used instead.